// base.proto
syntax = "proto3";

package base;

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LogoutRequest {
    string user_id = 1;
}

message LoginResponse {
    string user_id = 1;
    string status = 2;
}

message LogoutResponse {
    string user_id = 1;
    string status = 2;
}